### DISCUSSION TITLE
Squeezelite Devices (Muse Luxe) Display

### DISCUSSION THREAD

**Discussion by bluenazgul** [music-assistant/support#1305] [Feature requests and ideas]
In my Muse Luxe with external Display (should be Same as "Muse Mn Cast" there is Missing Time in Display on Standby and Missing Volume Info in Change Volume

**Comment by varac** [comment]
I just got a [Raspiaudio Muse Radio](https://raspiaudio.com/product/muse-radio/) with internal display, and it only shows an IP, nothing else.
In the [installation guide](https://forum.raspiaudio.com/t/muse-radio-squeezelite/1099) it shows how to use the Logitech media server ESP32 plugin to configure the screen. 
Now I am wondering if sth similar could be integrated into MA ? It would be great to make use of the display, i.e. showing the current song playing etc.
